style.css

```diff
@@ -532,3 +533,255 @@ input[type="submit"]:disabled {
 form#f2 {
   display: none;
 }
+
+/* start of desktop version design */
+@media only screen and (min-width: 768px) {
+  * {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+
+  nav {
+    display: flex;
+    justify-content: space-around;
+    width: 100%;
+    font-size: 20px;
+    height: 72px;
+    padding-top: 10px;
+  }
+
+  nav > img {
+    display: none;
+  }
+
+  nav ul {
+    display: flex;
+    justify-content: center;
+    width: 50%;
+    padding-top: 7px;
+  }
+
+  nav ul li {
+    list-style-type: none;
+    margin: 10px;
+  }
+
+  nav ul li a {
+    color: #fff;
+    font-size: 15px;
+    text-decoration: none;
+  }
+
+  #headline {
+    width: 100%;
+    height: 100vh;
+    background-image: url("Resources/Images/Illustration-header.png");
+    background-size: 47%;
+    background-position: 100%;
+    background-position-y: 10px;
+  }
+
+  #headline > ul {
+    position: relative;
+    float: left;
+    top: -21rem;
+    gap: 0.8rem;
+    flex-direction: column;
+  }
+
+  #headline > #mainhead {
+    font-size: 55px;
+    margin-top: 20px;
+    width: 80%;
+    margin-left: 80px;
+    line-height: 4rem;
+  }
+
+  #headline > .subhead {
+    font-size: 25px;
+    margin-top: 5px;
+    width: 80%;
+    margin-left: 80px;
+  }
+
+  #headline > p {
+    font-size: 16px;
+    width: 55%;
+    margin-top: 5px;
+    margin-left: 80px;
+    line-height: 2rem;
+  }
+
+  .headtext {
+    display: flex;
+    width: 100%;
+    height: 250px;
+    padding-left: 30px 20px;
+  }
+
+  #work > .first-box > .headtext > h2 {
+    border-bottom: none;
+    font-family: 'Inter', sans-serif;
+    font-size: 30px;
+    font-weight: 500;
+    text-align: center;
+    position: relative;
+    max-width: 600px;
+    margin-left: 10px;
+  }
+
+  /* to position hr */
+  .first-box > .ohh {
+    display: block;
+    position: relative;
+    top: -9.5rem;
+    left: 16rem;
+    width: 60%;
+    background-color: #212121;
+  }
+
+  .imgsection {
+    display: flex;
+    flex-direction: row;
+    height: 300%;
+  }
+
+  #work > .first-box > .imgsection > img {
+    width: 60%;
+    height: 523px;
+  }
+
+  .ttwork {
+    height: 19%;
+    width: 40%;
+  }
+
+  #work > .carttt {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+  }
+
+  #about {
+    background-image: url("Resources/Images/illustration-aboutme.png");
+    background-size: 50%;
+    background-position-y: 85px;
+  }
+
+  #about > p {
+    width: 50%;
+  }
+
+  .carttt > .cart button {
+    display: none;
+  }
+
+  #about > button {
+    background: #f55800;
+    border: none;
+    font-family: 'Inter', sans-serif;
+    color: var(--txtcolor);
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+    letter-spacing: 0.03em;
+    width: 10rem;
+  }
+
+  #about > .line {
+    width: 100%;
+  }
+
+  .listing {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+  }
```

Comment:
- [ ] Please 🙏🏽 pay attention to the suggerences from the last code reviewer:

> Kindly make sure to make all the changes from [here](https://www.loom.com/share/76c98f42d74846389b37e6769731224b).

- [ ] Kindly correct the elements in red to be similar to the Figma template 👇🏽 
![image](https://user-images.githubusercontent.com/58642949/187586668-17279d69-fa6b-4d47-8f0c-7d41f487ea63.png)